examples/2d/elixir_euler_vortex_split_shockcapturing.jl

```diff
@@ -0,0 +1,67 @@
+
+using OrdinaryDiffEq
```

Comment:
Similar here, I think we can lose the `_split` in the file name.

Reply:
Maybe rename it to `elixir_euler_vortex_uniform_shockcapturing.jl`, such that it is clear that it is the counterpart to `elixir_euler_vortex_mortar_shockcapturing.jl`?

Reply:
I am not a fan of this because for consistency we would always need to say uniform for non mortar/amr tests

Reply:
OK, fair enough